Marlin/Configuration_adv.h

```diff
@@ -800,7 +800,7 @@
 // For debug-echo: 128 bytes for the optimal speed.
 // Other output doesn't need to be that speedy.
 // :[0, 2, 4, 8, 16, 32, 64, 128, 256]
-#define TX_BUFFER_SIZE 0
+#define TX_BUFFER_SIZE 32
```

Comment:
This as well?

Reply:
Same as previous.

Reply:
Good good - after a bit of investigating this should remove some delay that's created by our M105 polling, but I have a branch in the works that will remove that altogether to keep the serial lines less cluttered. 👍